app/components/mapbox/basic-map.js

```diff
@@ -14,7 +14,7 @@ export default class MapboxBasicMapComponent extends Component {
   init(...args) {
     super.init(...args);
 
-    registerWaiter(this._mapIsLoaded);
+    // registerWaiter(this._mapIsLoaded);
```

Comment:
Can you explain what this was doing and why it's ok to delete it? In general, prefer deleting over commenting out.